cypress/integration/check_summary/summary-count-affecting-other-endpoints.spec.js

```diff
@@ -0,0 +1,59 @@
+/// <reference types="cypress" />
+
+Cypress.on('uncaught:exception', (err, runnable) => {
+  // returning false here prevents Cypress from
+  // failing the test
+  return false
+})
+
+describe('Testing summary count after deleting one of the endpoints', () => {
+  let enabledAttributes = null
+  let enabledClusters = null
+  it('create two different endpoints', () => {
+    cy.fixture('baseurl').then((data) => {
+      cy.visit(data.baseurl)
+    })
+    cy.fixture('data').then((data) => {
+      cy.addEndpoint(data.endpoint1)
+      cy.addEndpoint(data.endpoint2)
+    })
+  })
+  it("get second endpoint's summary and delete first endpoint", () => {
+    cy.fixture('data').then((data) => {
```

Comment:
In other steps of the test, we just get the count of the enabled summary and assign it to a variable.
This test is just to make sure the values are the same after deleting one of the endpoints and checking on other steps doesn't add any value because we have to check if a value equals itself.